### PR TITLE
Add organization switcher button

### PR DESCRIPTION
**Pull request by samkotlove**
Screenshot 2024-02-29 at 1 00 00 PM
Screenshot 2024-02-29 at 1 03 00 PM

### CODE REVIEW

**Comment by samkotlove** [comment]
Didn't test that a user with only one organization doesn't see the button

File: packages/react/src/provider.tsx [hunk 1]
```diff
@@ -2920,6 +2922,10 @@ export const PhotonProvider = (opts: {
     client.setOrganization(organizationId);
   };
 
+  const removeOrganization = () => {
+    client.removeOrganization();
+  };
+
```
Comment (samkotlove) [inline packages/react/src/provider.tsx:2928]: not sure if i like this, was also thinking setOrganization can take an optional parameter and leaving it out clears the organization

**Comment by jknapp25** [comment]
nit: here's a similar icon in the feather flavor
image

**Review by jknapp25 — APPROVED** [review]

File: apps/app/src/views/components/Nav.tsx [hunk 2]
```diff
@@ -49,15 +50,24 @@ export const Nav = () => {
...
   const onLogout = useCallback(() => {
     localStorage.removeItem('previouslyAuthed');
     logout({ returnTo: orgSettings.returnTo, federated: orgSettings.federated });
   }, [logout, orgSettings]);
 
+  const onSwitchOrganization = useCallback(() => {
+    removeOrganization();
+    login({
+      organizationId: undefined
+    });
```
[the remark this answers is not in the document]
Comment (samkotlove) [reply apps/app/src/views/components/Nav.tsx:68]: Yeah you're probably right (though I think switchOrganization is different, that would be if you know the organization you want)

**Comment by samkotlove** [comment]
https://www.loom.com/share/f5d3279f2cc246bd89e63ab6a691fe8b

**Review by marisbest2 — APPROVED** [review]
Nice

**Review by captDaylight — APPROVED** [review]

**Pull request merged**